DigitalLearningSolutions.Data.Tests/Services/SessionDataServiceTests.cs

```diff
@@ -139,13 +139,28 @@ public void UpdateSessionDuration_Should_Not_Update_Inactive_Session()
 
                 // When
                 const int sessionId = 468;
-                sessionDataService.UpdateSessionDuration(sessionId);
+                sessionDataService.UpdateDelegateSessionDuration(sessionId);
 
                 // Then
                 var updatedSessions = sessionTestHelper.GetCandidateSessions(candidateId);
 
                 updatedSessions.Should().BeEquivalentTo(startingSessions);
             }
         }
+
+        [Test]
+        public void StartAdminSession_Should_Start_Users_New_Admin_Session()
+        {
+            using (new TransactionScope())
+            {
+                // When
+                var adminUser = UserTestHelper.GetDefaultAdminUser();
+                var sessionId = sessionDataService.StartAdminSession(adminUser.Id);

```

Comment:
Do we need the "default admin user" rather than using an id of some sort directly?

Reply:
The AdminSessions table has the AdminID as a foreign key, so will only work if the ID used can be found in the database. My thinking here was that we already have an admin that we know is going to be in the database, so I might as well grab that and use it, rather than specifying an ID manually. Manually setting the ID might cause the test to break if the default user is changed about in the future on the database. Is there any harm to how the test is set up currently?

Reply:
In every test we write, we should assume that all the data in the database backup is there, so any existing admin id will do. I feel like calling the adminUser builder here gives this test an extra layer of complication that can be avoided.